addons/website/models/ir_qweb.py

```diff
@@ -106,12 +107,20 @@ def _prepare_frontend_environment(self, values):
 
         return irQweb
 
+    def _is_static_node(self, el, compile_context):
+        if el.tag in self.URL_ATTRS and not el.attrib.get('data-no-post-process'):
+            return False
+        return super()._is_static_node(el, compile_context)
+
     def _post_processing_att(self, tagName, atts):
         if atts.get('data-no-post-process'):
             return atts
 
         atts = super()._post_processing_att(tagName, atts)
 
+        if tagName not in self.URL_ATTRS:
+            return atts
+
```

Comment:
I didn't see this part when I purpose this change.
```py
atts = self._adapt_style_background_image(atts, website.get_cdn_url)`
```
Effectively, in multi website when they are one or more cdn the box with a dynamic background are not static. You can adapt the `_is_static_node` method for this.
For the performance, It's a very small impact because it's only when the template are computed into the cached function.